packages/loopring_v3/contracts/impl/BasicDepositContract.sol

```diff
@@ -115,15 +116,15 @@ contract BasicDepositContract is IDepositContract, ReentrancyGuard, Claimable
         override
         payable
         onlyExchange
-        nonReentrant
+        //nonReentrant
```

Comment:
Never, because only the exchange can call these functions and the exchange is always nonReentrant. Basically the deposit contract is like an upgradeable library. Removing this saves ~3,000 gas per call into the deposit contract.